packages/genesys-spark-components/src/components/stable/gux-time-picker/gux-time-picker.service.ts

```diff
@@ -22,13 +24,59 @@ export function getTimeDisplayValues(
       ? ['12', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11']
       : Array.from(Array(24).keys()).map(x => String(x).padStart(2, '0'));
 
-  return hourOptions.reduce((acc, hourOption) => {
+  const hourOptionsFormatted = hourOptions.reduce((acc, hourOption) => {
     return acc.concat(
       minuteOptions.map(
         minuteOption => `${hourOption}:${minuteOption}`
       ) as GuxISOHourMinute[]
     );
   }, [] as GuxISOHourMinute[]);
+
+  return clockType === '24h'
```

Comment:
@katie-bobbe-genesys this is where I check if the clocktype is 24h before applying the min/max check.